tools/src/launcher/Main.hs

```diff
@@ -133,7 +139,15 @@ optionsParser = do
         short   'w' <>
         help    "An argument to be passed to the wallet frontend executable." <>
         metavar "ARG"
+    loWalletLogging <- switch $
+        long    "wlogging" <>
+        help    "Bool that determines if wallet should log to stdout"
 
+    loWalletLogPath <- optional $ textOption $
+        long    "wallet-log-path" <>
+        help    ("File where wallet stdout/err will be redirected " <>
+                 "(def: temp file).") <>
```

Comment:
Why temp file? We discarded the idea of using temp files for logging. Also you should update the description of `--node-log-path` option.

Reply:
Will do, this was copy/pasta